### ISSUE TITLE
Maximizing doesn't work on Windows 10 

### ISSUE THREAD

**Issue by Kax675** [GregVido/mica-electron#42]
# Maximizing issue

Maximizing doesn't work even if `maximizable` is set to `true`

![image](https://github.com/user-attachments/assets/42ed77c3-9cdb-4379-b5cc-7356a353ed31)

The button at titlebar shows as normal but doesn't do anything if I click. I can see the maximize option greyed out if I right-click the titlebar.

The problem still occurs even if I do not set any effects in Windows 10

OS: `Windows 10 21H2 19044`

```
    mainWindow = new MicaBrowserWindow({
        width: 1196,
        height: 752,
        minWidth: 1000,
       titleBarStyle: 'hidden',
        titleBarOverlay: {
            color: 'rgba(0, 0, 0, 0)',
            symbolColor: nativeTheme.shouldUseDarkColors ? '#fff' : '#000',
        },
        minHeight: 600,
        frame: false,
        autoHideMenuBar: true,
        show: false,
        webPreferences: {
            preload: path.join(__dirname, 'Preload.js'),
            additionalArguments: [`--colors=${JSON.stringify(colors)}`, `--version=${app.getVersion()}`]
        }

    });

    mainWindow.loadURL('http://192.168.1.74:1931');
    mainWindow.setAutoTheme()
    if (IS_WINDOWS_11) {
        mainWindow.setMicaEffect();
    } else {
       mainWindow.setCustomEffect(WIN10.BLURBEHIND, '#000000', 0.7);
    }

    mainWindow.webContents.once('dom-ready', () => {
       mainWindow.show();
    });
```

**Comment by GregVido** [comment]
I'm going to download an ISO of Windows 10 tonight, I'll look at it tomorrow and let you know.

**Comment by Kax675** [comment]
> I'm going to download an ISO of Windows 10 tonight, I'll look at it tomorrow and let you know.

Thanks, let me know if you need further details 😀

**Comment by GregVido** [comment]
Normally I just released a version with the problem solved (1.5.16)

**Comment by Kax675** [comment]
Looks like it's fixed. Thanks!
